### DISCUSSION TITLE
add custom css to form

### DISCUSSION THREAD

**Discussion by manassingh7** [Budibase/budibase#5839] [Help]
i am unable to add custom css to my form . 

.spectrum-FieldLabel {
    text-overflow: clip;
    white-space: normal;
    padding-left: 5px;
    padding-right: 5px;
}
![css ankush](https://user-images.githubusercontent.com/52330720/168006790-ca2ff997-c240-4a91-ba39-f3486c6af37d.png)

**Comment by poirazis** [comment]
this isnt yet possible. you can only declare inline css styling rules in the custom CSS field. you cannot target elements with selectors, as the generated code is inline CSS and the declarations are injected inline in the wrapper element.

**Comment by dannyshin76** [comment]
Although the custom CSS is largely restricted, instead you can add an Embed element with "<style>" block to achieve pretty much everything you want. Just don't forget to add "!important" to custom styles in it. Otherwise they get ignored.

**Comment by melohagan** [comment]
Thank you @poirazis @dannyshin76 

I've added a docs page here to include your answers: https://docs.budibase.com/docs/custom-css
